start.py

```diff
@@ -394,6 +404,27 @@ def cm_server_not_dead(primary_node):
                                kerberos_principals=args.kerberos_principals)
 
 
+def _hdfs_namednode_format(deployment, cluster_name, primary_node):
```

Comment:
call this `_format_hdfs_namenodes` (same feedback as earlier)

Reply:
Done.

Reply:
Done.